js/grades.js

```diff
@@ -261,8 +255,8 @@ var fetchQueue = [];
                                 hasHandledGradeEdit = true;
                             };
                         }
-                        editGradeImg.addEventListener("click", createEditListener(assignment, gradeWrapper.parentElement, category, periods[0], gradeAddEditHandler));
-                        gradeWrapper.appendChild(editGradeImg);
+                        const gradeText = assignment.querySelector(".awarded-grade") || assignment.querySelector(".no-grade");
+                        gradeText.addEventListener("click", createEditListener(assignment, gradeWrapper.parentElement, category, periods[0], gradeAddEditHandler));

```

Comment:
Same thing here, keep the click handler on the image in addition to adding the new handler